src/server/attr_recov_db.c

```diff
@@ -78,73 +70,74 @@ extern int resc_access_perm;
 extern struct attribute_def	svr_attr_def[];
 extern struct attribute_def	que_attr_def[];
 
+
 /**
  * @brief
- *	Create a svrattrl structure from the attr_name, and values
- *
- * @param[in]	attr_name - name of the attributes
- * @param[in]	attr_resc - name of the resouce, if any
- * @param[in]	attr_value - value of the attribute
- * @param[in]	attr_flags - Flags associated with the attribute
+ *	Compute and check whether the quick save area has been modified
  *
- * @retval - Pointer to the newly created attribute
- * @retval - NULL - Failure
- * @retval - Not NULL - Success
+ * @param[in]	qs  - pointer to the quick save area
+ * @param[in]	len - length of the quick save area
+ * @param[in]   oh  - pointer to a opaque value of current quick save area signature/hash
```

Comment:
Please rename oh->oldhash. Also it looks like this also acts as OUT parameter. Can you please update that.